core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/Status.java

```diff
@@ -67,7 +67,8 @@ public enum Status {
     LOAD_BALANCER_UPDATE_FAILED(StatusKind.FINAL),
     UPGRADE_CCM_IN_PROGRESS(StatusKind.PROGRESS),
     UPGRADE_CCM_FAILED(StatusKind.FINAL),
-    UPGRADE_CCM_FINISHED(StatusKind.FINAL);
+    UPGRADE_CCM_FINISHED(StatusKind.FINAL),
+    DETERMINE_BACKUP_DATA_SIZES_IN_PROGRESS(StatusKind.PROGRESS);
```

Comment:
Try to avoid the keyword "BACKUP" in the enums.
With this change, we are just gathering the data size of datalake.